src/components/slide.js

```diff
@@ -10,20 +10,51 @@ const SlideContainer = styled('div')`
   width: ${({ theme }) => theme.size.width || 1366}px;
   height: ${({ theme }) => theme.size.height || 768}px;
   overflow: hidden;
+  ${({ autoLayout }) =>
+    autoLayout &&
+    css`
+      height: 100%;
+      display: flex;
+      flex-direction: column-reverse;
```

Comment:
We use `column-reverse` because the template is higher up in the DOM order because in non-auto layout mode it's an overlapping div and we don't want it to obstruct slide content.